src/main/scala/com/box/castle/router/kafkadispatcher/messages/KafkaDispatcherMessage.scala

```diff
@@ -79,3 +79,5 @@ private[kafkadispatcher] case class FetchOffsetKafkaResponse(response: OffsetRes
 
 private[kafkadispatcher] case class InternalTopicMetadataResponse(response: TopicMetadataResponse) extends KafkaResponse
 
+private[kafkadispatcher] case class UnknownTopicPartition[T <: DispatchToKafka](topicAndPartition: TopicAndPartition) extends KafkaDispatcherCommonMessage
```

Comment:
This sould just be

`private[kafkadispatcher] case class UnknownTopicPartition(topicAndPartition: TopicAndPartition) extends KafkaDispatcherCommonMessage`

Reply:
Done